### DISCUSSION TITLE
Skiovox Breakout-Need-Help-

### DISCUSSION THREAD

**Discussion by CGRmags** [3kh0/ext-remover#967] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
SO in the Skiovox breakout thing in discord, i have everything up to the part where i need to evaluate a payload to overwrite the extension. SO basically i need a payload to overwrite securely

**Comment by CGRmags** [comment]
@AshtonDavies help pls

**Reply by CGRmags** [reply]
@AshtonDavies help

**Reply by AshtonDavies** [reply]
I don't have payloads for that, sorry.

**Reply by CGRmags** [reply]
darn do  you know anyone that does?

**Reply by AshtonDavies** [reply, marked as answer]
> darn do  you know anyone that does?

Can you ask for help in the SKIOVOX breakout channel in Titanium Network?

https://discord.com/channels/419123358698045453/1175947388154810449

**Comment by CGRmags** [comment]
sure

On Mon, Mar 11, 2024 at 12:21 PM Ashton Davies ***@***.***>
wrote:

> darn do you know anyone that does?
>
> Can you ask for help in the SKIOVOX breakout channel in Titanium Network?
>
> https://discord.com/channels/419123358698045453/1175947388154810449
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/3kh0/ext-remover/discussions/967#discussioncomment-8750665>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BDXUJBDSBD2E2TALLVYXFJTYXYG3DAVCNFSM6AAAAABENKSG6CVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DONJQGY3DK>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

-- 
NOTICE: This email message and/or its attachments may contain information 
that is confidential or restricted. It is intended only for the individuals 
named as recipients in the message. If you are NOT an authorized recipient, 
you are prohibited from using, delivering, distributing, printing, copying, 
or disclosing the message or content to others and must delete the message 
from your computer. If you have received this message in error, please 
notify the sender by return email.
